pkg/backporter/backporter.go

```diff
@@ -0,0 +1,292 @@
+    </form>
+  </div>
+</nav>
+<div class="container">
+`
+
+const htmlPageEnd = `
+</div>
+<footer>
+<p class="small">Source code for this page located on <a href="https://github.com/openshift/ci-tools">GitHub</a></p>
+</footer>
+</body>
+</html>
+`
+
+const clonesTemplateConstructor = `
+	<h2 id="bugid"> <a href = "#bugid"> {{.Bug.ID}}: {{.Bug.Summary}} </a> | Status: {{.Bug.Status}} </h2>
+	<p> Target Release: {{ .Bug.TargetRelease }} </p>
+	{{ if .PRs }}
+		<p> GitHub PR: 
+		{{ if .PRs }}
+			{{ range $index, $pr := .PRs }}
+				{{ if $index}}|{{end}} 
+				<a href="{{ $pr.Type.URL }}/{{ $pr.Org}}/{{ $pr.Repo}}/pull/{{ $pr.Num}}">{{ $pr.Org}}/{{ $pr.Repo}}#{{ $pr.Num}}</a>
+			{{ end }}
+		{{ end }}
+		</p>
+	{{ else }}
+		<p> No linked PRs! </p>
+	{{ end }}
+	{{ if ne .Parent.ID .Bug.ID}}
+		<p> Cloned From: <a href = "/getclones?ID={{.Parent.ID}}"> Bug {{.Parent.ID}}: {{.Parent.Summary}}</a> | Status: {{.Parent.Status}}
+	{{ else }}
+		<p> Cloned From: This is the original! </p>
+	{{ end }}
+	<h4 id="clones"> <a href ="#clones"> Clones</a> </h4>
+	<table class="table">
+		<thead>
+			<tr>
+				<th title="Targeted version to release fix" class="info">Target Release</th>
+				<th title="ID of the cloned bug" class="info">Bug ID</th>
+				<th title="Status of the cloned bug" class="info">Status</th>
+				<th title="PR associated with this bug" class="info">PR</th>
+			</tr>
+		</thead>
+		<tbody>
+		{{ if .Clones }}
+			{{ range $clone := .Clones }}
+				<tr>
+					<td style="vertical-align: middle;">{{ $clone.TargetRelease }}</td>
+					<td style="vertical-align: middle;"><a href = "/getclones?ID={{$clone.ID}}">{{ $clone.ID }}</a></td>
+					<td style="vertical-align: middle;">{{ $clone.Status }}</td>
+					<td style="vertical-align: middle;">
+						{{range $index, $pr := $clone.PRs }}
+							{{ if $index}},{{end}}
+							<a href = "{{ $pr.Type.URL }}/{{$pr.Org}}/{{$pr.Repo}}/pull/{{$pr.Num}}" target="_blank"> {{$pr.Org}}/{{$pr.Repo}}#{{$pr.Num}}</a>
+						{{end}}
+					</td>
+				</tr>
+			{{ end }}
+		{{ else }}
+			<tr> <td colspan=4 style="text-align:center;"> No clones found! </td></tr>
+		{{ end }}
+		</tbody>
+	</table>`
+
+var (
+	clonesTemplate = template.Must(template.New("clones").Parse(clonesTemplateConstructor))
+	emptyTemplate  = template.Must(template.New("empty").Parse("{{.}}"))
+)
+
+// HandlerFuncWithErrorReturn allows returning errors to be logged
+type HandlerFuncWithErrorReturn func(http.ResponseWriter, *http.Request) error
+
+// Class to hold the UI data for the clones page
+type clonesTemplateData struct {
+	Bug    *bugzilla.Bug          //bug details
+	Clones []*bugzilla.Bug        // List of clones for the bug
+	Parent *bugzilla.Bug          // Root bug if it is a a bug, otherwise holds itself
+	PRs    []bugzilla.ExternalBug // Details of linked PR
+}
+
+//Writes an HTML page, prepends header in htmlPageStart and appends header from htmlPageEnd around tConstructor.
```

Comment:
nit: comments should have whitespace between `//` and the text (multiple times in this PR)

Reply:
yup. fixing that now